README.md

```diff
@@ -0,0 +1,161 @@
+# Capsule sandbox
+
+The Capsule sandbox is a containerized development environment for building [Capsule](https://github.com/capsule-rs/capsule) applications in Rust.
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Getting Started](#getting-started)
+  * [Vagrant and Docker](#vagrant-and-docker)
+  * [Linux Distributions](#linux-distributions)
+  * [Without Docker](#without-docker)
+  * [Kernel NIC Interface](#kernel-nic-interface)
+* [Packaging for Release](#packaging-for-release)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Introduction
+
+`Capsule` is built on Intel's [Data Plane Development Kit](https://www.dpdk.org/) release 18.11. While it is written in Rust, it needs to be able to call `DPDK`'s C functions installed separately as shared dynamic libaries. To make developing and running Capsule applications easy, we created the sandbox container with all the necessary dependencies. You can pull down the container and start writing network functions in stable `Rust` right away.
+
+## Getting Started
+
+To run the [sandbox](https://hub.docker.com/repository/docker/getcapsule/sandbox), the docker host must run on a Linux distribution. `DPDK` requires either Linux or FreeBSD. We plan to add FreeBSD support in the future.
+
+### Vagrant and Docker
+
+The quickest way to start the sandbox is to use the VirtualBox virtual machine included in this repository. If you are developing on either MacOS or Windows, this is the only way to write a `Capsule` application.
+
+You will need [`Vagrant`](https://www.vagrantup.com/) and [`VirtualBox`](https://www.virtualbox.org/) installed on the host with the following `Vagrant` plugins,
```

Comment:
after reviewing https://github.com/capsule-rs/capsule/pull/69/files#diff-04c6e90faac2675aa89e2176d2eec7d8R80, if we can remove the use of `you` as much as possible, I think it reads better (aka comment near https://github.com/capsule-rs/capsule/pull/69/files#diff-04c6e90faac2675aa89e2176d2eec7d8R32).